src/EnergyPlus/DataZoneEquipment.hh

```diff
@@ -190,7 +190,7 @@ namespace DataZoneEquipment {
 
         // Default Constructor
         EquipMeterData()
```

Comment:
Initialize in declaration, remove explicit default constructor. If you touch an object, you should do this.